src/coreclr/nativeaot/System.Private.Reflection.Execution/src/Internal/Reflection/Execution/ReflectionDomainSetupImplementation.cs

```diff
@@ -14,19 +14,12 @@ namespace Internal.Reflection.Execution
     //=========================================================================================================================
     internal sealed class ReflectionDomainSetupImplementation : ReflectionDomainSetup
     {
-        public ReflectionDomainSetupImplementation(ExecutionEnvironmentImplementation executionEnvironment)
+        public ReflectionDomainSetupImplementation()
```

Comment:
We can delete these three lines too - the C# compiler will make this constructor for us.

Reply:
I am following these [design guidelines](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/constructor):
✔️ DO explicitly declare the public parameterless constructor in classes, if such a constructor is required.

Thank you for helping with this issue!

Reply:
FWIW, I don't think we follow that rule in the runtime repo.

Can't quickly find a reference, but this is the [closest one](https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/breaking-change-rules.md#behavioral-changes).

✗ Disallowed
Adding a constructor to a class which previously had no constructor, without also adding the default constructor

Reply:
The design guidelines apply to public APIs only. They do not apply to internal implementation details (the case here). 

The internal implementation details are covered by https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md that is a lot less prescriptive than the design guidelines.